### DISCUSSION TITLE
Save the page completely with a shortcut and add also a shorcut for the preview

### DISCUSSION THREAD

**Discussion by Soulgear99** [elementor/elementor#24313] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

When I attempt to save the page using a shortcut, it doesn't save completely. However, when I preview the page, it does save. Yet, upon returning to the original page, the changes are not preserved. 

https://github.com/elementor/elementor/assets/70857278/c19f7492-19d5-4e4f-b365-489997a0eede

### Describe the solution you'd like

It will be nice to completely save with a shortcut without clicking 'Publish'.  Could you please consider adding a shortcut for previewing the page? This would enhance practicality.

**Comment by GuidoEijrond** [comment]
cmd-s works for me to save a page (without publishing)
shortcut for publishing or previewing would be nice! 👍
